workflows/nextflow.config

```diff
@@ -20,6 +20,9 @@ profiles{
         errorStrategy = { task.exitStatus in 137..140 ? 'retry' : 'terminate' }
         maxRetries = 1
       }
+      withLabel: bigdisk {
+        queue = 'nextflow-batch-bigdisk-queue'
```

Comment:
Where do we get `nextflow-batch-bigdisk-queue` from? Is this the name of a batch queue that exists in AWS?

Reply:
That is correct. This is part of the AWS config in our organization: You can see the available queues at https://console.aws.amazon.com/batch/v2/home?region=us-east-1#

They are defined in the terraform files in the aws directory, but I wouldn't worry about trying to understand those...

Reply:
Gotcha!